internal/server/iigointernal/monitoring.go

```diff
@@ -60,13 +65,28 @@ func (m *monitor) evaluateCache(roleToMonitorID shared.ClientID, ruleStore map[s
 	return performedRoleCorrectly
 }
 
-func (m *monitor) findRoleToMonitor(roleAccountable baseclient.Client) (shared.ClientID, baseclient.Role) {
+func (m *monitor) findRoleToMonitor(roleAccountable baseclient.Client) (shared.ClientID, shared.Role) {
 	switch roleAccountable.GetID() {
 	case m.speakerID:
-		return m.presidentID, baseclient.President
+		return m.presidentID, shared.President
 	case m.presidentID:
-		return m.judgeID, baseclient.Judge
+		return m.judgeID, shared.Judge
 	default:
-		return m.speakerID, baseclient.Speaker
+		return m.speakerID, shared.Speaker
```

Comment:
always speaker when the others don't fit?

Reply:
Yes, there are only 3 possible cases for calling this func (from President, Speaker or Judge within orchestration.go which is server side) so i didn't know whether it was better to do 3 cases or 2 cases and a default :/

Reply:
3 cases + default & error is the best practice :)

Reply:
okie dokie, i've changed it!